plugins/modules/tmp_vcenter_services_info.py

```diff
@@ -0,0 +1,40 @@
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+import ansible_collections.ansible.vmware_rest.plugins.module_utils.vmware_httpapi as vmware_httpapi
+
+
+def main():
+    argument_spec = vmware_httpapi.VmwareRestModule.create_argument_spec()
+    argument_spec.update({
+    'service': dict(type="str", required=True),
+    })
+    module = vmware_httpapi.VmwareRestModule(
+        argument_spec=argument_spec,
+        supports_check_mode=True,
+        is_multipart=True,
+        #use_object_handler=True,
+    )
+
+    
+    url = "/rest" + "/vcenter/services/{id}".format(**module.params)
```

Comment:
This will raise a Keyerror since there is no `id` in `module.params`

Reply:
:smiley: Good catch, I will fix that.